packages/gcp/src/__tests__/fixtures/googleapis.fixtures.ts

```diff
@@ -1,130 +1,113 @@
 /*
  * © 2021 Thoughtworks, Inc.
  */
-import { compute_v1 } from 'googleapis'
-import Schema$Instance = compute_v1.Schema$Instance
-import Schema$MachineType = compute_v1.Schema$MachineType
-import Schema$Image = compute_v1.Schema$Image
-import Schema$InstanceAggregatedList = compute_v1.Schema$InstanceAggregatedList
-import Schema$Disk = compute_v1.Schema$Disk
-import Schema$DiskAggregatedList = compute_v1.Schema$DiskAggregatedList
-import Schema$AddressAggregatedList = compute_v1.Schema$AddressAggregatedList
-import Schema$Address = compute_v1.Schema$Address
-
-export type InstanceData = {
-  data: Schema$Instance
-}
 
-type MachineTypeData = {
-  data: Schema$MachineType
-}
+import { ScopedListResult } from '../../lib/ServiceWrapperTypes'
+import { google } from '@google-cloud/compute/build/protos/protos'
+import IDisk = google.cloud.compute.v1.IDisk
+import IInstance = google.cloud.compute.v1.IInstance
+import IMachineType = google.cloud.compute.v1.IMachineType
+import IImage = google.cloud.compute.v1.IImage
+import IAddress = google.cloud.compute.v1.IAddress
 
-type ImageDetails = {
-  data: Schema$Image
+interface IterableMockResponse {
+  next(): Promise<IteratorResult<ScopedListResult>>
 }
 
-type DiskData = {
-  data: Schema$Disk
-}
-
-type AddressDetails = {
-  data: Schema$Address
-}
-
-type InstanceAggregatedList = {
-  data: Schema$InstanceAggregatedList
-}
-
-type DiskAggregatedList = {
-  data: Schema$DiskAggregatedList
+const mockAsyncIterator = {
+  // * = fancy generator function (https://javascript.info/generators)
```

Comment:
pretty cool, didn't know about generators

Reply:
All the cool kids are doing it these days